app/src/components/transactions/message-view/StakeTxDetails.vue

```diff
@@ -4,14 +4,26 @@
     <div class="tx__content__left">
       <h3>{{ caption }}</h3>
       <span>With&nbsp;</span>
-      <router-link :to="`/staking/validators/${transaction.details.to[0]}`">
+      <router-link
+        :to="{
+          name: `validator`,
+          params: {
+            validator: transaction.details.to[0],
+            networkId: transaction.networkId,
+          },
+        }"
+      >
         <img
           v-if="validator && validator.picture"
           :src="validator.picture"
           class="validator-image"
           :alt="`validator logo for ` + validator.name"
         />
-        {{ transaction.details.to[0] | resolveValidatorName(validators) }}
+        {{
+          transaction.details.to[0]
+            | formatAddress
```

Comment:
this looks weird. `resolveValidatorName` already does a `formatAddress`

Reply:
true, I am going to check that

Reply:
ok, fixed